### ISSUE TITLE
Dont print debug output on delete

### ISSUE THREAD

**Issue by bprashanth** [kubernetes/kubernetes#30946]
Farily new cluster:
```
$ kubectl delete rc nginx-ingress-controller
W0818 17:22:43.648289   11034 request.go:351] Field selector: v1 - replicationControllers - metadata.name - nginx-ingress-controller: need to check if this is versioned correctly.
replicationcontroller "nginx-ingress-controller" deleted
```
With yaml
```yaml
# nginx ingress controller RC
apiVersion: v1
kind: ReplicationController
metadata:
  name: nginx-ingress-controller
  labels:
    k8s-app: nginx-ingress-lb
spec:
  replicas: 1
  selector:
    k8s-app: nginx-ingress-lb
  template:
    metadata:
      labels:
        k8s-app: nginx-ingress-lb
        name: nginx-ingress-lb
    spec:
      terminationGracePeriodSeconds: 0
      containers:
      - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
        name: nginx-ingress-lb
        imagePullPolicy: Always
        livenessProbe:
          httpGet:
            path: /healthz
            port: 10254
            scheme: HTTP
          initialDelaySeconds: 30
          timeoutSeconds: 5
        # use downward API
        env:
          - name: POD_NAME
            valueFrom:
              fieldRef:
                fieldPath: metadata.name
          - name: POD_NAMESPACE
            valueFrom:
              fieldRef:
                fieldPath: metadata.namespace
        ports:
        - containerPort: 80
          hostPort: 80
        - containerPort: 443
          hostPort: 443
        # we expose 18080 to access nginx stats in url /nginx-status
        # this is optional
        - containerPort: 18080
          hostPort: 18080
        args:
        - /nginx-ingress-controller
        - --default-backend-service=kube-system/default-http-backend
```
Unsure if it's just rcs or everything.

**Comment by AdoHe** [comment]
@bprashanth [here](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/restclient/request.go#L350) ptal.

**Comment by lavalamp** [comment]
Isn't there already a change out to fix this?

This happens when you forget to put in a field filter...

**Comment by AdoHe** [comment]
This has been fixed in #30950, feel free to close this one.
